options/locale/locale_en-US.ini

```diff
@@ -2151,21 +2154,23 @@ settings.protected_branch.delete_rule = Delete Rule
 settings.protected_branch_can_push = Allow push?
 settings.protected_branch_can_push_yes = You can push
 settings.protected_branch_can_push_no = You cannot push
-settings.branch_protection = Branch Protection for Branch '<b>%s</b>'
+settings.branch_protection = Branch Protection Rules for Branch '<b>%s</b>'
 settings.protect_this_branch = Enable Branch Protection
 settings.protect_this_branch_desc = Prevents deletion and restricts Git pushing and merging to the branch.
 settings.protect_disable_push = Disable Push
 settings.protect_disable_push_desc = No pushing will be allowed to this branch.
 settings.protect_enable_push = Enable Push
 settings.protect_enable_push_desc = Anyone with write access will be allowed to push to this branch (but not force push).
+settings.protect_enable_merge = Enable Merge
+settings.protect_enable_merge_desc = Anyone with write access will be allowed to merge the pull requests into this branch.
 settings.protect_whitelist_committers = Whitelist Restricted Push
 settings.protect_whitelist_committers_desc = Only whitelisted users or teams will be allowed to push to this branch (but not force push).
 settings.protect_whitelist_deploy_keys = Whitelist deploy keys with write access to push.
 settings.protect_whitelist_users = Whitelisted users for pushing:
 settings.protect_whitelist_search_users = Search users…
 settings.protect_whitelist_teams = Whitelisted teams for pushing:
 settings.protect_whitelist_search_teams = Search teams…
-settings.protect_merge_whitelist_committers = Enable Merge Whitelist
+settings.protect_merge_whitelist_committers = Whitelist Restricted Merge
```

Comment:
Hmm the old translation sounded a bit more clear to me. "Whitelist Restricted Merge" doesn't roll of the tongue so well and might even confuse translators.

Reply:
```suggestion
settings.protect_merge_whitelist_committers = Enable Merge Whitelist
```
Then revert